docs/v1.1/README.md

```diff
@@ -0,0 +1,136 @@
+# Instamojo PHP API [![Latest Stable Version](https://poser.pugx.org/instamojo/instamojo-php/v/stable)](https://packagist.org/packages/instamojo/instamojo-php) [![License](https://poser.pugx.org/instamojo/instamojo-php/license)](https://opensource.org/licenses/MIT)
+
```

Comment:
Add the seamless js checkout piece to this too. So that developers still on the old version can still benefit